.github/workflows/build-m1-binaries.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - nightly
+      - main
```

Comment:
Why do we want to execute this on main ? Since you already compiling vision inside test-m1.yml.

Reply:
The `test-m1` only tests on py3.8 and executes only part of the checks (no wheel validation). I thought it would be best to do this at least once in each PR on main to be able to catch the issues easier. It's a middle ground tradeoff between executing the tests in every PR/commit and doing so only once a day. Hopefully the volume of commits that get merged every day on TorchVision isn't significant to affect the overall resources.